### ISSUE TITLE
Optional Tags become Subjects

### ISSUE THREAD

**Issue by johancc** [johancc/CovEd#21]
Whichever tag you choose under optional tags ends up being displayed under subjects. 
![Annotation 2020-04-13 223050](https://user-images.githubusercontent.com/45345485/79197014-e91a0200-7de5-11ea-882a-3f6d304c073f.png)

**Comment by oupadhyay** [comment]
I tried to replicate the issue. I ran the website and registered. Each of my tags went to the correct field. Is there any other details on how you got this?

![image](https://user-images.githubusercontent.com/46334571/79369044-3345d500-7f05-11ea-911d-f8de9e45e84b.png)

**Comment by johancc** [comment]
After you clicked submit, did they appear under the optional tags field?

**Comment by oupadhyay** [comment]
After clicking submit, I accidentally submitted it. Sorry! But after checking, I can not see them switched and they stayed in the optional tags. Should I send you a video of how it looks on my end? I can't seem to replicate the switching of tags and subjects.

**Comment by johancc** [comment]
Huh might just be an issue on my end. Let me double check it and if not I'll close the issue

**Comment by johancc** [comment]
How I replicated the bug, on branch master:
1) Make a new test account
2) Register, and go to the profile page
3) Select a new optional tag. The subjects will be overridden by whatever new optional tag you choose.
4) Saving works, but the optional tag will be saved as a subject.

**Comment by oupadhyay** [comment]
I see it now. 
When you first register, it has no problem, but afterward, when we go back to edit the profile, it doesn't let us add tags and clears the subjects. Ok got it. I will look into it.

**Comment by oupadhyay** [comment]
I have tried to look through the code, but I do not see a field in the tutee object or (`Const`) that stores the tags and only a field for the subjects. Where would the tags field be stored even if a user inputted them? 

Moreover, looking at the HTML code, I see that both fields point to the same object, which might be the cause of the issue. I have shown this in the video below. I could not attach videos so I had to do a Vimeo link. 
[Vimeo Link](https://vimeo.com/408207383)

I think it is an issue of not having a field to store the tags in the tutees. While there was a commit to add it in the tutors it has not been done in the tutee.js.
